apps/evm/lib/evm/gas.ex

```diff
@@ -382,14 +384,14 @@ defmodule EVM.Gas do
 
   def operation_cost(
         :staticcall,
-        [gas_limit, to_address, value, _in_offset, _in_length, _out_offset, _out_length],
+        [gas_limit, to_address, _in_offset, _in_length, _out_offset, _out_length],
         _machine_state,
         exec_env
       ) do
     to_address = Address.new(to_address)
 
-    Configuration.call_cost(exec_env.config) + call_value_cost(value) +
-      new_account_cost(exec_env, to_address, value) + gas_limit
+    Configuration.call_cost(exec_env.config) + new_account_cost(exec_env, to_address, 0) +
```

Comment:
What is that `0` being passed? Could we extract that into a variable and give it a name? Otherwise it's a little bit of a mystery number.